packages/lab/src/combo-box-next/ComboBoxNext.tsx

```diff
@@ -247,7 +246,12 @@ export const ComboBoxNext = forwardRef(function ComboBoxNext<T>(
         role="combobox"
         variant={variant}
         value={inputValue}
-        {...getTriggerProps()}
+        ref={reference}
```

Comment:
If this is needed for floating ui to work, should it be at least `triggerRef`?
Or `inputRef` should already make it work?

Reply:
yep is needed for floating-ui to work, before this was stated in trigger ref but to incorporate the event handlers into floating-ui i've stated it here instead

Reply:
so the ref needs to be attached to the input - so will change to trigger ref